### ISSUE TITLE
Challenge 13: Fatal Error

### ISSUE THREAD

**Issue by tolik518** [evanbowman/skyland-beta#25]
Upon starting Challenge 13, the game instantly crashes with a fatal error (error should be present on all recent builds, with clean savegame). 

```common-lisp
[ERR: Wrong number of arguments passed to function: '(flag-show (player) 0)]
```

**Comment by evanbowman** [comment]
How did this break? Explanation: I forgot to add one of the challenge scripts to git version control. I was working on a feature and temporarily reverted the changes, but the revert did not apply to the script because it wasn't tracked.

**Comment by tolik518** [comment]
> How did this break? Explanation: I forgot to add one of the challenge scripts to git version control. I was working on a feature and temporarily reverted the changes, but the revert did not apply to the script because it wasn't tracked.

Oooh, I was already wondering how you wrote the script so fast when looking at the changes.

**Comment by evanbowman** [comment]
Yeah, so I had this idea that I would display different flag graphics for different varieties of enemies. So I added a second argument to an existing script function called flag-show, which currently tells the engine to display a flag for a castle. The second argument would tell the game which version of flag graphics to display for an opponent. I still plan to add the feature at a later date. All I did to resolve the issue was search for `(flag-show (player) 0)` in the codebase, and found that a script file wasn't tracked, and deleted the second argument
